plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -297,39 +302,36 @@ static void build_command_line(struct ffmpeg_muxer *stream, struct dstr *cmd,
 		num_tracks++;
 	}
 
-	dstr_init_move_array(cmd, os_get_executable_path_ptr(FFMPEG_MUX));
-	dstr_insert_ch(cmd, 0, '\"');
-	dstr_cat(cmd, "\" \"");
-
-	dstr_copy(&stream->path, path);
```

Comment:
This was actually used in a bunch of places outside build_command_line for printing the destination path, now starting ffmpeg_muxer outputs just says its recording to null.

Reply:
Ah that's an oversight, I guess we still need to copy to `&stream->path`, should be an easy fix at least.